src/org/rust/cargo/icons/CargoIcons.kt

```diff
@@ -0,0 +1,8 @@
+package org.rust.cargo.icons
+
+import com.intellij.openapi.util.IconLoader
+
+public object CargoIcons {
+    public val ICON = IconLoader.getIcon("/org/rust/lang/icons/cargo.png")
+    public val LOCK_ICON = IconLoader.getIcon("/org/rust/lang/icons/cargo-lock.png")
```

Comment:
You mean align subsequent field declarations:
public val ICON             = IconLoader ....
public val LOCK_ICON = IconLoader ...

In java it could be done, but kotlin formatter is not capable yet (at time I last checked)